extension/src/experiments/workspace.ts

```diff
@@ -314,7 +314,7 @@ export class WorkspaceExperiments extends BaseWorkspaceWebviews<
     )
 
     experiments.dispose.track(
-      experiments.onDidChangeColumnOrder(() => {
+      experiments.onDidChangeColumnOrderOrStatus(() => {
```

Comment:
Since an `onDidChangeColumnOrder` and `onDidChangeColumnStatus` event does the same thing, I combined the two events into one. Though, we could split them into two events if preferred :)